api/token.ts

```diff
@@ -1,20 +1,32 @@
 import { getAddress } from 'viem'
 
 import { getBuiltGraphSDK } from '../.graphclient'
-import { Currency } from '../utils/currency'
+import { Asset } from '../model/asset'
 
-const { Tokens } = getBuiltGraphSDK()
+const { getAssets } = getBuiltGraphSDK()
 
-export async function fetchTokens(): Promise<Currency[]> {
-  const { tokens } = await Tokens()
+export async function fetchAssets(): Promise<Asset[]> {
+  const { assets } = await getAssets(
+    {},
+    // {
+    //   cacheOptions: { ttl: 60 * 60 * 24 },
+    // },
```

Comment:
I believe the usage of `cacheOptions` might not be functional, as I couldn't locate a corresponding `cacheOptions` variable within the graph-client repository. Would you consider referencing this example for guidance? Here's the link: [https://github.com/Urigo/graphql-mesh/tree/master/examples/openapi-react-weatherbit](https://github.com/Urigo/graphql-mesh/tree/master/examples/openapi-react-weatherbit)

Reply:
Let's use `responseCache` plugin

1. https://github.com/Urigo/graphql-mesh/blob/master/examples/openapi-react-weatherbit/getDailyForecastByCoordinates.query.graphql
2. https://github.com/Urigo/graphql-mesh/blob/master/examples/openapi-react-weatherbit/.meshrc.yml